pylsp/config/config.py

```diff
@@ -98,6 +98,10 @@ def __init__(self, root_uri, init_opts, process_id, capabilities):
                 self._plugin_settings, plugin_conf
             )
 
+        self._plugin_settings = _utils.merge_dicts(
+            self._plugin_settings, self._init_opts.get("pylsp", {})
```

Comment:
Language servers typically don't scope initialization settings to server name because, unlike the workspace settings, those are specific to the server itself and don't have any chance of conflicting with other settings.

Reply:
I am happy to address this, but it seems that there is lack of consensus on whether `initializationOptions` should be used for server configuration.

Reply:
`pylsp` is the options namespace used for this server, so I'm fine with using it.